ripd/ripd.c

```diff
@@ -2764,9 +2783,51 @@ static int rip_update_jitter(unsigned long time)
 	return jitter / JITTER_BOUND;
 }
 
+/* It should only be used when processing incoming info update from zebra.
+ * Other situations, it is not sufficient to lookup the rip instance by
+ * vrf_name only without using the instance number.
+ */
+static struct rip *rip_lookup_by_name(const char *vrf_name)
+{
+	if (vrf_name == NULL && rip_global && rip_global->vrf_id == VRF_DEFAULT)
+		return rip_global;
+	if (vrf_name && rip_global && rip_global->name &&
+	    0 == strcmp(vrf_name, rip_global->name))
+		return rip_global;
+	return NULL;
+}
+
+static struct rip *rip_cmd_lookup_rip(struct vty *vty,
+				      struct cmd_token *argv[],
+				      const int argc, int create)
```

Comment:
bool create?  I'd like to start using bool's for when we are clearly using boolean values instead of int's